apps/cli/src/commands-index.ts

```diff
@@ -102,4 +102,4 @@ export default {
   user,
   version,
   walletconnect
-};
+} as any;
```

Comment:
Please do not use `any`.

Reply:
is this one really a problem? it's requiring a type annotation due to references in the lib. But it's the CLI, will we enforce this rule even in the CLI?

Reply:
Done